sites/docs/docs/components/sankey-diagram.md

```diff
@@ -29,6 +34,52 @@ hide_table_of_contents: false
 />
 ```
 
+## Multi-level
+
+The syntax for multi-level sankey diagrams is the same, but the 
+underlying query must represent all the levels using the same 
+`sourceCol` and `targetCol`, so it is necessary to `union`
+ each level together.  `sourceCal` nodes on the next level will be linked to `targetCol` nodes in the previous level with the same name.  
+
+For example, here is the source for the visuals above.
+
+```markdown
+```sql traffic_source
+select 'google' as source, 'all_traffic' as target, 100 as count
+union all
+select 'direct' as source, 'all_traffic' as target, 50 as count
+union all
+select 'facebook' as source, 'all_traffic' as target, 25 as count
+union all
+select 'bing' as source, 'all_traffic' as target, 25 as count
+union all
+select 'tiktok' as source, 'all_traffic' as target, 25 as count
+union all
+select 'twitter' as source, 'all_traffic' as target, 25 as count
+union all
+select 'linkedin' as source, 'all_traffic' as target, 25 as count
+union all
+select 'pinterest' as source, 'all_traffic' as target, 25 as count
+union all
+select 'all_traffic' as source, '/' as target, 50 as count
+union all
+select 'all_traffic' as source, '/docs' as target, 150 as count
+union all
+select 'all_traffic' as source, '/blog' as target, 25 as count
+union all
+select 'all_traffic' as source, '/about' as target, 75 as count
+‍```
```

Comment:
I suspect this example is a bit confusing because it uses many union statements when in reality if you have an actual DB connection rather than generating fake data with SQL, you only need to to one union per extra level in the sankey:

Perhaps this is a better example?

```sql
select 
    channel as source,
    'all_traffic' as target
    count(user_id) as count
from web_events
group by 1,2

union all

select 
    'all_traffic' as source,
    page_route as target
    count(user_id) as count
from web_events
group by 1, 2
```